webrender/src/device.rs

```diff
@@ -1169,7 +1202,34 @@ impl Device {
             }
         }
 
-        gl::bind_framebuffer(gl::FRAMEBUFFER, self.default_fbo);
+        // TODO(gw): Hack! Modify the code above to use the normal binding interfaces the device exposes.
+        gl::bind_framebuffer(gl::READ_FRAMEBUFFER, self.bound_read_fbo.0);
```

Comment:
I think the old code was just *wrong* but happened to not show up any bugs thus far. All the tests pass with this change, so it seems correct to me to make this behave the way I'd expect the semantics to work :)